src/test/java/com/openlattice/rehearsal/SetupTestData.kt

```diff
@@ -36,42 +36,73 @@ open class SetupTestData : MultipleAuthenticatedUsersBase() {
+            loginAs("admin");
+            val tokenAdmin = AuthenticationTest.getAuthentication(authOptions).credentials
+            val email = getUserInfo(SetupEnvironment.admin).email
+
+            return arrayOf(
+                    "-${ShuttleCliOptions.ENVIRONMENT}=LOCAL",
+                    "-${ShuttleCliOptions.FETCHSIZE}=1000",
+                    "-${ShuttleCliOptions.UPLOAD_SIZE}=1000",
+                    "-${ShuttleCliOptions.TOKEN}=$tokenAdmin",
+                    "-${ShuttleCliOptions.CREATE}=$email")
         }
+
         /**
-         * Import datasets via Shuttle
-         * @param
+         * Import datasets from CSV via Shuttle
+         * @param flightFileName
+         * @param dataFileName
          */
-        fun importDataSet(flightFileName: String, dataFileName: String) {
-            loginAs("admin")
-            val tokenAdmin = AuthenticationTest.getAuthentication(authOptions).credentials
-
+        fun importDataSetFromCSV(flightFileName: String, dataFileName: String) {
             val flightFile = File(Thread.currentThread().contextClassLoader.getResource(FLIGHT_FOLDER).file,
                     flightFileName).absolutePath
             val dataFile = File(Thread.currentThread().contextClassLoader.getResource(DATA_FOLDER).file, dataFileName)
                     .absolutePath
-            val email = getUserInfo(SetupEnvironment.admin).email
+            val shuttleArgs : Array<String> = getDefaultShuttleArgs()
 
-            main(arrayOf(
-                    "-${ShuttleCliOptions.FLIGHT}=$flightFile",
-                    "-${ShuttleCliOptions.CSV}=$dataFile",
-                    "-${ShuttleCliOptions.ENVIRONMENT}=LOCAL",
-                    "-${ShuttleCliOptions.TOKEN}=$tokenAdmin",
-                    "-${ShuttleCliOptions.CREATE}=$email"))
+            main(shuttleArgs.plus(
+                    arrayOf(
+                            "-${ShuttleCliOptions.FLIGHT}=$flightFile",
+                            "-${ShuttleCliOptions.CSV}=$dataFile"
+                    )
+            ))
         }
 
+        /**
+         * Import datasets from CSV via Shuttle
+         * @param flightFileName
+         * @param dataConfiguration
+         * @param dataConfigurationKey
+         */
+        fun importDataSetFromAtlas(flightFileName: String, dataConfiguration: String, dataConfigurationKey: String) {
+            val flightFile = File(Thread.currentThread().contextClassLoader.getResource(FLIGHT_FOLDER).file,
+                    flightFileName).absolutePath
+            val configFile = File(Thread.currentThread().contextClassLoader.getResource(CONFIG_FOLDER).file, dataConfiguration)
+                        .absolutePath
+            val shuttleArgs : Array<String> = getDefaultShuttleArgs()
+            main(shuttleArgs.plus(
+                    arrayOf(
+                            "-${ShuttleCliOptions.FLIGHT}=$flightFile",
+                            "-${ShuttleCliOptions.SQL}=${FLIGHT_SQL}",
```

Comment:
One minor last thing to consider: if we want to have multiple imports from atlas with different queries, I would consider making `FLIGHT_SQL` also an input and not a fixed thing.